lib/routemaster/config.rb

```diff
@@ -24,6 +24,16 @@ def cache_redis
       RedisBroker.instance.get(ENV.fetch('ROUTEMASTER_CACHE_REDIS'))
     end
 
+    def hosts
```

Comment:
Bit of functional magic in there, so maybe worth leaving a comment on whatthe resulting data structure looks like?